src/app/pages/details/common/links-to-translations.tsx

```diff
@@ -45,14 +45,11 @@ function AnotherLanguage({
         [translations, locale]
     );
 
-    if (!translation) {
-        return null;
-    }
-
     // translation is guaranteed to have a valid value, because the locale
     // is pulled from translations
     return (
-        <a href={`/details/books/${translation.slug}`}>
+         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+         <a href={`/details/books/${translation!.slug}`}>
```

Comment:
The issue is that TS knows that `find` might return `undefined`. But it won't (as the comment notes).
I can use a `?` instead of a `!` if it's less alarming. It doesn't require the eslint-disable that way.